lib/config.js

```diff
@@ -7,6 +7,9 @@ var bool = require('normalize-bool')
 var trunc = require('unicode-byte-truncate')
 
 config.INTAKE_STRING_MAX_SIZE = 1024
+config.CAPTURE_LOCATION_STACK_TRACE_OFF = 'off'
+config.CAPTURE_LOCATION_STACK_TRACE_NON_ERRORS = 'non-errors'
```

Comment:
@roncohen what do you feel about this value? I don't like that it consists of two words and contains a hyphen. There's a high likelihood that people will mistype it. Though right now it's the default value, which means people probably don't need to type it 🤔

The only alternative I've been able to come up with is `log`.

Reply:
`non-errors` is a bit ambiguous. I thought it was for `spans`. Can we come up with a better name? I don't think the dash is a problem generally.

Reply:
`log.*` and `exception.*` is only allowed on the `/v1/errors` endpoint of the APM Server. This PR is about what happens when you call `agent.captureError()` and is not related to any performance instrumentation

Reply:
> This PR is about what happens when you call agent.captureError() and is not related to any performance instrumentation

@watson I understand. I'm saying that from the choice of wording for `CAPTURE_LOCATION_STACK_TRACE_NON_ERRORS` and `non-errors` its not clear that this is unrelated to performance instrumentation.

Reply:
It makes sense in context of the `captureError` function as you can pass it both errors and things that are not errors. But I don't like it either. Would like a better name if you have any suggestions?

Reply:
Something like `always`, `never` and `when-missing-location` maybe?

Reply:
After talking with @roncohen we came to the conclusion that the config option should be renamed `captureErrorLogStackTraces` and the options should be `never`, `messages` and `always`